homework/week4/15651.py

```diff
@@ -0,0 +1,17 @@
+n, m = map(int, input().split())
```

Comment:
이 문제는 product를 사용하면 좋을 것 같습니다.
```suggestion
from itertools import product

n, m = map(int, input().split())

for nums in product(list(range(1, n + 1)), repeat=m):
    print(*nums)

```